egs2/spring_speech/local/data.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+SECONDS=0
+
+lang=
+release_name="SPRING_INX"
+
+log "$0 $*"
+. utils/parse_options.sh
+
+. ./db.sh
+. ./path.sh
+. ./cmd.sh
+
+stage=1
+stop_stage=100000
+
+lang=${lang^}
+echo Language : ${lang}
+
+data_url="https://asr.iitm.ac.in/SPRING_INX/data/${release_name}_${lang}"
+
+if [ $# -ne 0 ]; then
+    log "Error: No positional arguments are required."
+    exit 2
+fi
+
+if [ -z "${release_name}" ]; then
+    log "Fill the value of '${release_name}' of db.sh"
```

Comment:
I feel this log msg is incorrect. Could you double check?

Reply:
Yes, I will correct it. Thanks.